### PR TITLE
Added zodiac signs #33

### PR DESCRIPTION
**Pull request by adityagi02**

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Make sure to follow the formatting guidelines in #5 otherwise the dictionary won't work

You can update the branch with the changes

Let me know if I can help

**Comment by adityagi02** [comment]
done!!

**Comment by DenverCoder1** [comment]
Please note that in order for the shortcuts to work, there must be a single tab character before and after each symbol as explained in the issue and no colons. Please check the formatting of the other lines. You may copy the tab symbols from the other lines to ensure it is correct.

Also note that `\taurus` (`♉`) and `\cancer` (`♋`) have already been added to the dictionary (see lines 933-934)  so these lines can be removed so there are no duplicates.

Let me know if you have any questions.

File: dictionary.txt [hunk 1]
```diff
@@ -913,12 +937,9 @@ _rp	₎
 \frac56	⅚	
 \frac18	⅛	
 \frac38	⅜	
-\frac58	⅝	
+\frac58 ⅝ 
 \frac78	⅞	
 \co	℅	
--	−
```
Comment (DenverCoder1) [inline dictionary.txt:919]: Put this line back, it's a shortcut for converting hyphen (`-`) to longer minus sign (`−`)

File: dictionary.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 \Iota	I	
 \lnot	¬	
 \neg	¬	
-^2	²	
+^2  ²
```
Comment (DenverCoder1) [inline dictionary.txt:5]: Notice that there are **tab** characters here. You removed them and put spaces.

There are supposed to only be **tab** characters.

image

```suggestion
^2	²	
```

File: dictionary.txt [hunk 3]
```diff
@@ -913,12 +937,9 @@ _rp	₎
 \frac56	⅚	
 \frac18	⅛	
 \frac38	⅜	
-\frac58	⅝	
+\frac58 ⅝
```
Comment (DenverCoder1) [inline dictionary.txt:940]: You removed the tabs

```suggestion
\frac58	⅝	
```

File: dictionary.txt [hunk 4]
```diff
@@ -896,11 +896,35 @@ _eq	₌
...
+\venus ♀ 
+\earth ⊕ 
+\mars ♂ 
+\ceres ⚳ 
+\jupiter ♃ 
+\saturn ♄ 
+\uranus ♅ 
+\neptune ♆ 
+\pluto  ♇ 
+\frac12 ½
```
Comment (DenverCoder1) [inline dictionary.txt:927]:
```suggestion
\hslash	ℏ	
\dagger	†	
\ddagger	‡	
\smalltilde	˜	
\aries	♈︎	
\taurus	♉︎	
\gemini	♊︎	
\cancer	♋︎	
\leo	♌︎	
\virgo	♍︎	
\libra	♎︎	
\scorpio	♏︎	
\sagittarius	♐︎	
\capricorn	♑︎	
\aquarius	♒︎	
\pisces	♓︎	
\sun	☉	
\moon	☾	
\mercury	☿	
\venus	♀	
\earth	⊕	
\mars	♂	
\ceres	⚳	
\jupiter	♃	
\saturn	♄	
\uranus	♅	
\neptune	♆	
\pluto	♇	
\frac12	½	
```

File: dictionary.txt [hunk 5]
```diff
@@ -900,6 +900,30 @@ _rp	₎
...
+\mercury	☿	
+\venus	♀	
+\earth	⊕	
+\mars	♂	
+\ceres	⚳	
+\jupiter	♃	
+\saturn	♄	
+\uranus	♅	
+\neptune	♆	
+\pluto	♇
```
Comment (DenverCoder1) [inline dictionary.txt:926]:
```suggestion
\pluto	♇	
-	−	
```

**Comment by DenverCoder1** [comment]
Looks great! Thanks for the contribution :tada:

**Pull request merged**